components/package-lock.json

```diff
@@ -0,0 +1,5334 @@
+{
```

Comment:
This is `npm`'s lock file, right? I think we can delete it since we're using `yarn` in this repo. If so, let's add it to the `.gitignore`.

Reply:
Ah yes, good spot, thanks.